Sources/SwiftDiagnostics/FixIt.swift

```diff
@@ -46,3 +46,48 @@ public struct FixIt {
     self.changes = changes
   }
 }
+
+public extension FixIt {
+  var edits: [SourceEdit] {
+    var existingEdits = [SourceEdit]()
+    for change in changes {
+      let edit = edit(from: change)
+      var isOverlapping = false
+      for existingEdit in existingEdits {
+        // The edit overlaps with the previous edit. We can't apply both
+        // without conflicts. Apply the one that's listed first and drop the
+        // later edit.
+        if edit.range.overlaps(existingEdit.range) {
```

Comment:
[Question] the ranges of edits in a FixIt wouldn't overlap with each other, but the range of edits in multiple FixIts still possibly overlap each other. i was wondering if it's expected in our case.